web/server/vue-cli/package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "scripts": {
     "build": "rimraf dist && cross-env NODE_ENV=production webpack --config config/webpack.prod.js",
-    "server:codechecker": "CodeChecker server -w e2e/__codechecker/workspace > e2e/__codechecker/server.log",
+    "server:codechecker": "CodeChecker server --port 8002 -w e2e/__codechecker/workspace > e2e/__codechecker/server.log",
```

Comment:
What is happening here?

Reply:
Removed

Reply:
Checked again, this is already part of the repo. I just lazily added this instead of rebasing during development.